core/app/models/spree/fulfilment_changer.rb

```diff
@@ -105,19 +105,21 @@ def run_tracking_inventory
         # These two statements are the heart of this class. We change the number
         # of inventory units requested from one shipment to the other.
         # We order by state, because `'backordered' < 'on_hand'`.
+        # We start to move the new actual backordered quantity, so the remainder
```

Comment:
`remainder` -> `remaining`?

Reply:
Since you asked, I believe `remaining quantity` is _at_least_ clearer 👍  (IMO `remainder` is correct, as it's the remainder of the subtraction of the total quantity to move minus the backordered quantity).

Reply:
Ah, I thought it was a typo. If it's related to the subtraction, I think it's ok. Up to you how you want to leave it then.